sdk/automation/Azure.ResourceManager.Automation/src/Generated/Models/RunbookTypeEnum.cs

```diff
@@ -10,7 +10,10 @@
 
 namespace Azure.ResourceManager.Automation.Models
 {
-    /// <summary> Gets or sets the type of the runbook. </summary>
+    /// <summary>
+    /// Gets or sets the type of the runbook.
+    /// Serialized Name: RunbookTypeEnum
+    /// </summary>
     public readonly partial struct RunbookTypeEnum : IEquatable<RunbookTypeEnum>
```

Comment:
I don't think `Enum` is a good suffix here, maybe we should drop it.

Reply:
fixed